### PR TITLE
feat: add breadcrumbs component

### PR DESCRIPTION
**Pull request by div-Leo**
**What:**

​
**Why:**

Add new component Breadcrumbs
​
**How:**

Created new component
Added documentation
Wrote tests

**Media:**
Capture-2023-08-11-144009

Issue:  #324

### CODE REVIEW

File: src/components/Breadcrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+`;
+
+const BreadcrumbsListItem = styled.li`
+    display: flex;
+`;
+
+const Breadcrumbs = ({ children, inverted }: BreadcrumbsProps): JSX.Element => {
+    const arrayChildren = Children.toArray(children);
+
+    return (
```
Comment (nlopin) [inline src/components/Breadcrumbs/Breadcrumbs.tsx:45]: It is recommended to have the breadcrumb wrapped with the `nav` tag for a11y. 
I found a very good example on the [a11y style guide website](https://a11y-style-guide.com/style-guide/section-navigation.html#kssref-navigation-breadcrumbs) we can use as a markup example.

- [ ] follow the a11y recommendations

File: src/components/Breadcrumbs/docs/Breadcrumbs.storybook.mdx
Comment (nlopin) [inline src/components/Breadcrumbs/docs/Breadcrumbs.storybook.mdx:1]: How do we use `Breadcrumbs` with the react-router or next links? Let's show them examples

**Review by nlopin — CHANGES_REQUESTED** [review]
We need to add accessibility to the component.

Another question: can we make the clickable area bigger? At the moment only the text is clickable, why not having the whole block clickable? I think it can be achieved by wrapping the `li` with `a`.

Now:
CleanShot 2023-08-14 at 13 38 03@2x

Ideally:
CleanShot 2023-08-14 at 13 39 16@2x

File: src/components/Breadcrumbs/docs/Breadcrumbs.storybook.mdx
Comment (div-Leo) [reply src/components/Breadcrumbs/docs/Breadcrumbs.storybook.mdx:1]: We already have an example of how to use it with react-router, is that enough or do we want to add also Next Links?

**Comment by div-Leo** [comment]
@nlopin I couldn't wrap the `li` with the `a` because of the structure I chose, it was easier to use padding to increase the area, and the result should be okay.

Screenshot 2023-08-17 at 10 07 21

File: src/components/Breadcrumbs/docs/Breadcrumbs.storybook.mdx
Comment (nlopin) [reply src/components/Breadcrumbs/docs/Breadcrumbs.storybook.mdx:1]: I missed that, sorry

**Review by nlopin — APPROVED** [review]

**Pull request merged**